submissions/innazozulia/DOM practice/main.js

```diff
@@ -0,0 +1,153 @@
+  },
+  {
+    name: 'Metapod',
+    imgSrc: './img/Metapod.jpeg',
+    description: 'The shell covering this Pokémons body is as hard as an iron slab. Metapod does not move very much. It stays still because it is preparing its soft innards for evolution inside the hard shell.'
+  },
+  {
+    name: 'Butterfree',
+    imgSrc: './img/Butterfree.jpeg',
+    description: 'Butterfree has a superior ability to search for delicious honey from flowers. It can even search out, extract, and carry honey from flowers that are blooming over six miles from its nest.'
+  },
+  {
+    name: 'Weedle',
+    imgSrc: './img/Weedle.jpeg',
+    description: 'Weedle has an extremely acute sense of smell. It is capable of distinguishing its favorite kinds of leaves from those it dislikes just by sniffing with its big red proboscis (nose).'
+  },
+  {
+    name: 'Kakuna',
+    imgSrc: './img/Beedrill.jpeg',
+    description: 'Kakuna remains virtually immobile as it clings to a tree. However, on the inside, it is extremely busy as it prepares for its coming evolution. This is evident from how hot the shell becomes to the touch.'
+  },
+  {
+    name: 'Beedrill',
+    imgSrc: './img/Kakuna.jpeg',
+    description: 'Beedrill is extremely territorial. No one should ever approach its nest—this is for their own safety. If angered, they will attack in a furious swarm.'
+  },
+  {
+    name: 'Pidgey',
+    imgSrc: './img/Pidgey.jpeg',
+    description: 'Pidgey has an extremely sharp sense of direction. It is capable of unerringly returning home to its nest, however far it may be removed from its familiar surroundings.'
+  },
+  {
+    name: 'Rattata',
+    imgSrc: './img/Rattata.jpeg',
+    description: 'Rattata is cautious in the extreme. Even while it is asleep, it constantly listens by moving its ears around. It is not picky about where it lives—it will make its nest anywhere.'
+  },
+];
+
+const container = Document.createElement('div');
+container.classList.add('container');
+Document.body.appendChild(container);
+
+const pageName = Document.createElement('h1');
+pageName.classList.add('page-name');
+pageName.textContent = "Choose your Pokemon";
+container.appendChild(pageName);
+
+const character = Document.createElement('div');
+character.classList.add('dictionary-name');
+character.textContent = "The Character";
+container.appendChild(character);
+
+const divMain = Document.createElement('div');
+divMain.classList.add('main');
+container.appendChild(divMain);
+
+const divMenu = Document.createElement('div');
+divMenu.classList.add('menu');
+divMain.appendChild(divMenu);
+
+const navigation = Document.createElement('ul');
+navigation.classList.add('navigation');
+const nav = Document.querySelectorAll('li');
+divMenu.appendChild(navigation);
+
+pokemons.forEach((val, key) => {
+  const li = Document.createElement('li');
+  li.classList.add('list-element');
+  li.textContent = pokemons[key].name;
+  if(key == 0){
```

Comment:
No need to check it in the each loop iteration. You can add this class to the element after finishing rendering this pokemons array